nowastecrawler/ingredients_dict.json

```diff
@@ -0,0 +1 @@
uszone gruszki", "woda w miarę potrzeb", "rodzinny fix gulasz z warzywami knorr", "duży burak", "dorsz filet", "rodzinny fix spaghetti bolognese z warzywami knorr", "sałata maslowa", "nasiona słonecznika", "czerwona lub czarna fasola z puszki", "mała młoda marchew", "por (biała część)", "sos sałatkowy zioła ogrodowe z trybulą knorr", "dynia piżmowa", "świeże czerwone chili", "nasiona dyni", "czarna soczewica beluga", "bułki do burgerów", "sos hellmann's original bbq", "chili", "ogórki z sałatki szwedzkiej", "rzodkiewka", "łopatka jagnięca", "krojone pomidory z puszki", "gałązka tymianku", "zielona soczewica", "bulion jagnięcy lub inny", "ocet balsamiczny", "bazylia do dekoracji", "ligawa wołowa", "pomidory pelati", "strąk chili", "ser oscypek", "przyprawa do gulaszu knorr", "obrana dynia piżmowa", "odcedzona ciecierzyca z puszki", "limonka otarta ze skórki i wyciśnięty sok", "amarantus", "cebula cukrowa", "strąk papryki chili", "śmietana 18 %", "sera feta", "sok z połowy cytryny", "sos sałatkowy naturalnie smaczne truskawkowy knorr", "mała czerwona cebula", "ser mozzarella mini", "świeży tymianek kilka gałązek", "bataty", "kalafior świeży lub mrożony", "natka pieruszki (posiekana)", "puszka warzyw w stylu meksykańskim ( czerwona fasola, kukurydza , papryka)", "fix gulasz z kurczakiem knorr", "bułka namoczona w mleku", "pierś z indyka", "ananasy z puszki", "kukurydza mała puszka", "sok pomarańczowy", "grzanki", "kapusta włoska", "ser wędzony, starty", "suszone śliwki najlepiej wędzone", "udka z kurczaka bez skóry i kości", "suszone śliwki bez pestek", "mięso mielone np. wieprzowe", "ostra papryka", "czosnek, ząbek", "kolendra lub natka pietruszki", "średniej wielkości cebule", "świeżo mielony czarny pieprz", "garść posiekanej natki pietruszki", "mięso wieprzowe", "suszone, namoczone i posiekane grzyby", "mała, pokrojona w talarki marchewka", "fasola wymoczona przez noc", "mała, drobno posiekana cebula", "łopatka wieprzowa", "zielona fasolka", "kurczak wędzony", "cytryna do dekoracji", "mięso mielone wołowe", "ser pleśniowy", "kura pieczona (bez kości)", "zupa grzybowa knorr", "mrożone podgrzybki", "grzyby (prawdziwki lub kozaki)", "makaron muszelki", "ciepła woda", "ser biały", "tłuszcz", "pałki z kurczaka lub skrzydełka", "ciecierzyca z puszki lub świeżo ugotowana", "pasta harissa", "jogurt grecki", "kasza jaglana ugotowana", "tymianek świeży gałązka", "orzeszki pinii", "olej z suszonych pomidorów", "masło klarowane do smażenia", "wołowina np. karkówka", "szalotki", "czerwone wino, najlepiej pinot noir", "świeży rozmaryn, gałązka", "świeży tymianek, gałązka", "goździki z indonezji knorr", "por mały", "borowiki świeże", "podgrzybki suszone", "bulion warzywny lub drobiowy", "parmezan tarty", "majeranek świeży", "naturalnie smaczne do kurczaka w sosie pieczarkowym knorr", "groch", "kości lub boczek", "świeży szpinak", "fix kurczak po meksykańsku knorr", "nieduży kalafior", "czosnek z usa knorr", "cytyna", "mleko roślinne", "cukier brązowy", "kakao"]
```

Comment:
we need this json in git? I'm not sure

Reply:
I also wanted to point this out, but then I realized that this file is here for a while :laughing: